starboard/shared/starboard/player/filter/filter_based_player_worker_handler.h

```diff
@@ -46,6 +47,10 @@ class FilterBasedPlayerWorkerHandler : public PlayerWorker::Handler,
   FilterBasedPlayerWorkerHandler(
       const SbPlayerCreationParam* creation_param,
       SbDecodeTargetGraphicsContextProvider* provider);
+  FilterBasedPlayerWorkerHandler(
+      const StarboardExtensionMaxInputSizePlayerCreationParam*
+          starboard_extension_creation_param,
+      SbDecodeTargetGraphicsContextProvider* provider);
```

Comment:
We can always used the new enhanced creation param in our filter player, to reduce redundant code.

Reply:
Renamed it to `StarboardExtensionEnhancedPlayerCreationParam` to allow all extensions via this code path.